File: fizzBuzz.py
'''

'''

def FizzBuzz(num : int) -> str:
    rem3: Final[int] = num % 3 ## variables intented as const, warned by static check
    rem5: Final[int] = num % 5
    if rem3 == 0:
        if rem5 == 0:
            return('Fizzbuzz')
        else: #elif num % 5 != 0:
            return('Fizz')
    elif rem5 == 0:
        return('Buzz')
    else:
        return num

def main():
    print ('Hello World')
    print(f"{FizzBuzz(3)=}")
    print(f"{FizzBuzz(5)=}")
    print(f"{FizzBuzz(30)=}")
    print(f"{FizzBuzz(1)=}")
    for i in range(7):
        if 0 < i < 5: # possible limiting bounds
            print(f"Looped FizzBuzz({i}) = {FizzBuzz(i)}")
    
if __name__ == "__main__":
    # a=int(input('Enter a number: '))
    main()

File: fizzBuzz.cs
using System;

using System.Linq; // for Enumerable itaration
using System.Diagnostics; // time measurement

class HelloWorld {
  static void FizzBuzzOneline(int x) {
    int rem3 = x % 3;
    int rem5 = x % 5;
    Console.WriteLine($"{(rem3 == 0 ? "Fizz" : "")}{(rem5 == 0 ? "Buzz" : "")}{(rem3 != 0 && rem5 != 0 ? x.ToString() : "")}");
  static void FizzBuzzWide(int x) {
        //  const 
        bool rem3 = (0 == (x % 3));
        bool rem5 = (0 == (x % 5));
        if (rem3)
            {           
            if (rem5)
                Console.WriteLine("FizzBuzz");
            else
                Console.WriteLine("Fizz");
            }
        else if (rem5)
            Console.WriteLine("Buzz");
        else
            Console.WriteLine(x);
  }
  static void Main() {
    //Console.WriteLine("Hello World");
        // var  // works as auto type
        Stopwatch 
        stopwatch = Stopwatch.StartNew();
    Enumerable.Range(1, 16).ToList().ForEach(x =>
       {
         const int rem3 = x % 3;
         const int rem5 = x % 5;
    Console.WriteLine($"{(rem3 == 0 ? "Fizz" : "")}{(rem5 == 0 ? "Buzz" : "")}{(rem3 != 0 && rem5 != 0 ? x.ToString() : "")}");
    } );
        stopwatch.Stop();
        // Display elapsed time in different formats
        Console.WriteLine($"Elapsed Time: {stopwatch.Elapsed}"); // Full TimeSpan
        Console.WriteLine($"Elapsed Milliseconds: {stopwatch.ElapsedMilliseconds} ms");
        Console.WriteLine($"Elapsed Ticks: {stopwatch.ElapsedTicks} ticks");
  }
}
